<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerMainWindow</class>
 <widget class="QWidget" name="CustomerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 255);
color: rgb(0, 0, 0);
</string>
  </property>
  <widget class="QPushButton" name="transferButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>money transfer </string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewCardsButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>view Cards</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editCardButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>edit Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid red;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
